ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -579,7 +579,7 @@ protocolInfoCardano protocolParamsByron@ProtocolParamsByron {
     -- The major protocol version of the last era is the maximum major protocol
     -- version we support.
     maxMajorProtVer :: MaxMajorProtVer
-    maxMajorProtVer = MaxMajorProtVer (pvMajor protVerAlonzo)
+    maxMajorProtVer = MaxMajorProtVer (pvMajor protVerBabbage)
```

Comment:
Do we need to mention this in the changelog. It will affect donwstream consumers such as the `db-analyzer` (albeit positively)

Reply:
Oh right, I will add an entry :+1:

Reply:
I went with 

> The maximum major protocol version of the Cardano protocol is now correctly based off of Babbage instead of Alonzo.